CMakeLists.txt

```diff
@@ -46,9 +46,6 @@ include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Config.cmake)
 include(GNUInstallDirs)
 
 # setup version numbers
-set(VERSION_MAJOR 3)
-set(VERSION_MINOR 0)
-set(VERSION_PATCH 0)
 set(VERSION_IS_RELEASE OFF)
```

Comment:
This could be right after the `project` command to keep related things together.

Reply:
Good idea. I moved that up.